docs/tutorial/graphical.rst

```diff
@@ -223,30 +223,30 @@ For :class:`~pymeasure.display.plotter.Plotter`, you can make a sub-class that o
             # use logarithmic x-axis (e.g. for frequency sweeps)
             plot.setLogMode(x=True)
 
-For :class:`~pymeasure.display.windows.ManagedWindow`, the mechanism to customize plots is much more flexible by using specialization via inheritance. Indeed :class:`~pymeasure.display.windows.ManagedWindowBase` is the base class for :class:`~pymeasure.display.windows.ManagedWindow` and :class:`~pymeasure.display.windows.ManagedImageWindow` which are subclasses ready to use for GUI.
+For :class:`~pymeasure.display.windows.managed_window.ManagedWindow`, the mechanism to customize plots is much more flexible by using specialization via inheritance. Indeed :class:`~pymeasure.display.windows.managed_window.ManagedWindowBase` is the base class for :class:`~pymeasure.display.windows.managed_window.ManagedWindow` and :class:`~pymeasure.display.windows.managed_image_window.ManagedImageWindow` which are subclasses ready to use for GUI.
 
 Defining your own ManagedWindow's widgets
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The parameter :code:`widget_list` in :class:`~pymeasure.display.windows.ManagedWindowBase` constructor allow to introduce user's defined widget in the GUI results display area.
-The user's widget should inherit from :class:`~pymeasure.display.widgets.TabWidget` and could reimplement any of the methods that needs customization.
+The parameter :code:`widget_list` in :class:`~pymeasure.display.windows.managed_window.ManagedWindowBase` constructor allow to introduce user's defined widget in the GUI results display area.
+The user's widget should inherit from :class:`~pymeasure.display.widgets.tab_widget.TabWidget` and could reimplement any of the methods that needs customization.
 In order to get familiar with the mechanism, users can check the following widgets already provided:
 
-- :class:`~pymeasure.display.widgets.LogWidget`
-- :class:`~pymeasure.display.widgets.PlotWidget`
-- :class:`~pymeasure.display.widgets.ImageWidget`
+- :class:`~pymeasure.display.widgets.log_widget.LogWidget`
+- :class:`~pymeasure.display.widgets.plot_widget.PlotWidget`
+- :class:`~pymeasure.display.widgets.image_widget.ImageWidget`
 
 
 Using the sequencer
 ~~~~~~~~~~~~~~~~~~~
 
-As an extension to the way of graphically inputting parameters and executing multiple measurements using the :class:`~pymeasure.display.windows.ManagedWindow`, :class:`~pymeasure.display.widgets.SequencerWidget` is provided which allows users to queue a series of measurements with varying one, or more, of the parameters. This sequencer thereby provides a convenient way to scan through the parameter space of the measurement procedure.
```

Comment:
Thanks for fixing the links here!

Reply:
No problem, I should have caught that when I split out widgets and windows.